definitions/ext-vpc_network/summary_metrics.yml

```diff
@@ -27,6 +37,11 @@ egressBytes:
       from: KFlow
       where: "provider = 'kentik-vpc' AND flow_direction = 'egress'"
       eventId: entity.guid
+    aws-kinesis-firehose/vpc-flow-logs:
+      select: sum(bytes*sample_rate)
```

Comment:
Have we got test data that actually shows a sample_rate?  That is an artifact of the ktranslate lambda and will not exist in a pure kinesis firehose

Reply:
Yep - `sample_rate` will be added by us in the kinesis config. For the initial pass, we're just setting it to 1. I'm working on a lambda that will allow the customer to set the value and whatever they set will get passed along in the telemetry.

<img width="1335" alt="image" src="https://user-images.githubusercontent.com/6722433/155563496-f22fab12-bbca-401d-b75c-2fbedf11b757.png">